docs/sources/alerting/fundamentals/alert-rules/_index.md

```diff
@@ -120,6 +120,5 @@ When choosing which alert rule type to use, consider the following comparison be
 | Add expressions to transform<wbr /> your data and set alert conditions         | Yes                                                                                                                          | No                                                                                                                                                      |
 | Use images in alert notifications                                              | Yes                                                                                                                          | No                                                                                                                                                      |
 | Organization                                                                   | Organize and manage access with folders                                                                                      | Use namespaces                                                                                                                                          |
-| Evaluation of alert rules in the same evaluation group                         | Concurrent evaluation                                                                                                        | Sequential evaluation                                                                                                                                   |
```

Comment:
Now, there is no major difference in the evaluation engine between both alert rule types, so better to not display this in this comparison table.